modules/videoio/test/test_precomp.hpp

```diff
@@ -41,4 +41,25 @@ inline void generateFrame(int i, int FRAME_COUNT, cv::Mat & frame)
 #endif
 }
 
+class BunnyParameters
+{
+private:
+    int bunny_width, bunny_height, bunny_fps, bunny_count;
```

Comment:
It is not necessary to store values in the object:
```.cpp
class BunnyParameters {
public:
  inline static int getWidth() { return 672; }
  inline static getCount() { return cvRound(getFps() * getTime()); }
  ...

// call BunnyParameters::getCount()
```

Filename generation can be provided by this class too:
```.cpp
inline static String getFilename(const String &ext) { 
  return cvtest::TS::ptr()->get_data_path() + "video/big_buck_bunny" + ext; 
}
```